### PR TITLE
docs: update deploy pages's subtitles to make it clear and consistent

### PR DESCRIPTION
**Pull request by quentin7b**
## Status

READY

## Description

As discussed [here](https://github.com/VeryGoodOpenSource/dart_frog/pull/1173#discussion_r1448688467) the idea is to switch from bullet list to header bullet list

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [x] 📝 Documentation
- [ ] 🗑️ Chore

## Some examples

Here are the rendered indexes

Google Cloud Run
Screenshot 2024-01-13 at 16 18 37

AWS
Screenshot 2024-01-13 at 16 18 50

Digital Ocean
Screenshot 2024-01-13 at 16 19 17

Globe
Screenshot 2024-01-13 at 16 19 32

### CODE REVIEW

File: docs/docs/deploy/aws-app-runner.md [hunk 1]
```diff
@@ -72,19 +76,25 @@ docker build build \
 - `[REPOSITORY_NAME]`: The name of the repository you created earlier
 - `[IMAGE_TAG]`: A name given to this image to identify it in the repository
 
-This command will build the Docker image on your computer and can take a few seconds to a few minutes.
+It will build the Docker image on your computer, this can take a few seconds to a few minutes.
+
+### 3. Push the image to ECR
 
-3. Push the image to ECR by running:
+Now that you have the image on your computer, you can push it to ECR
```
Comment (wolfenrain) [inline docs/docs/deploy/aws-app-runner.md:83]:
```suggestion
Now that you have the image on your computer, you can push it to ECR:
```

**Review by wolfenrain — APPROVED** [review]
LGTM, one small suggestion

File: docs/docs/deploy/globe.md [hunk 2]
```diff
@@ -25,13 +25,17 @@ globe login
...
+### 1. Deploy your API for production
+
+In order to deploy, you can simply run:
 
 ```bash
 globe deploy --prod
 ```
 
-2. Congratulations! 🎉 You have successfully deployed your API to Globe. You can now access your API at the Deployment URL that is available through the Globe dashboard.
+### 5. Enjoy your API on Globe!
```
Comment (alestiago) [inline docs/docs/deploy/globe.md:36]:
```suggestion
### 2. Enjoy your API on Globe!

File: docs/docs/deploy/google-cloud-run.md [hunk 3]
```diff
@@ -65,7 +69,9 @@ Running this command will do three things:
 - Build the Docker image in [Cloud Build](https://cloud.google.com/build) and upload it to [Artifact Registry](https://cloud.google.com/artifact-registry)
 - Deploy the image to the specified Cloud Run service
 
-3. Congratulations! 🎉 You have successfully built and deployed your API to Cloud Run. You can now access your API at the Service URL that is printed in the last line of output.
+### 5. Enjoy your API on Cloud Run!
```
Comment (alestiago) [inline docs/docs/deploy/google-cloud-run.md:72]:
```suggestion
### 3. Enjoy your API on Cloud Run!
```

**Review by alestiago — DISMISSED** [review]

**Review by alestiago — APPROVED** [review]

**Review by wolfenrain — APPROVED** [review]

**Comment by alestiago** [comment]
Thank you @quentin7b for the contribution 💙 !

**Pull request merged**